### PR TITLE
[fix] Permissions for enigma plugin closes #21

### LINKED ISSUE #21: Access error for PGP keys
`Application error (600): Enigma plugin: Keys directory isn't writeable: /var/www/roundcube/plugins/enigma/home`
error when trying to access PGP Keys in the settings.

### PR DESCRIPTION
**Pull request by Rafi594**
## Problem
- *Description of why you made this PR*

## Solution
- *And how you fix that*

## PR Status
Work finished. Package_check, basic tests and upgrade from last version OK.  
Could be reviewed and tested.

## Validation
---
*Minor decision*
- [x] **Upgrade previous version** : JimboJoe
- [x] **Code review** : Maniack C
- [x] **Approval (LGTM)** : Maniack C
- [x] **Approval (LGTM)** : JimboJoe
- [x] **CI succeeded** : [Build Status/badge/icon)](https://ci-apps-dev.yunohost.org/jenkins/job/roundcube_ynh%20PR22%20(Official_fork)/)  
When the PR is mark as ready to merge, you have to wait for 3 days before really merge it.

### CODE REVIEW

File: scripts/install [hunk 1]
```diff
@@ -179,7 +179,7 @@ ynh_store_file_checksum "${final_path}/config/config.inc.php"
 #=================================================
 
 # Set permissions to app files
-chown -R root: "$final_path"
+chown -R $app: "$final_path"
```
Comment (JimboJoe) [inline scripts/install]: You should limit your owner change to the strict minimum, so to the enigma folder or plugins folder.
Furthermore, you should also handle the upgrade script as well 😉

**Comment by maniackcrudelis** [comment]
[Issue 21](https://github.com/YunoHost-Apps/roundcube_ynh/issues/21) says "directory isn't writeable: /var/www/roundcube/plugins/enigma/home"
So, it would be really better that only this directory became writeable.

**Comment by Rafi594** [comment]
Fixed

**Comment by JimboJoe** [comment]
Thanks @Rafi594, will be merged in 3 days.

**Pull request merged**